src/codecTools/toJsonSchema.ts

```diff
@@ -30,7 +30,7 @@ type MappableType =
  * @returns a JSON schema object
  * @see https://json-schema.org/understanding-json-schema/basics.html
  */
-export const toJsonSchema = (_type: any): JSONSchema7 => {
+export const toJsonSchema = (_type: any, strict = false): JSONSchema7 => {
```

Comment:
I think you can also do this with `t.exact` too? Although I think that does not throw an error, it just strips out any unknown props

Reply:
https://github.com/gcanti/io-ts/blob/master/index.md#exact-types

Reply:
yeah! I think even if we used exact, we'd still need to translate that to `additionalParameters` being specified.